docs/releases.md

```diff
@@ -5,9 +5,30 @@ A Conduit release has the following parts:
   * a file with checksums for the packages
   * a changelog
   * the source code
-* a GitHub package, which is the official Docker image for Conduit. It's available on GitHub's Container Registry.
+* a GitHub package, which is the official Docker image for Conduit. It's available on GitHub's Container Registry. The 
+latest Docker image which is not a nightly is tagged with `latest`.
 
-### How to release a new version?
+### How to release a new version
 A release is triggered by pushing a new tag which starts with `v` (for example `v1.2.3`). Everything else is then handled by 
-GoReleaser and GitHub actions. To push a new tag, please use the script [tag.sh](https://github.com/ConduitIO/conduit/blob/main/scripts/tag.sh), 
-which also checks if the version conforms to SemVer.
+GoReleaser and GitHub actions. To push a new tag, please use the script [scripts/tag.sh](https://github.com/ConduitIO/conduit/blob/main/scripts/tag.sh), 
+which also checks if the version conforms to SemVer. Example:
+```
+scripts/tag.sh 1.2.3
```

Comment:
Shouldn't this be `scripts/tag.sh v1.2.3`?

Reply:
It's actually 1.2.3 and then the script adds the prefix.